src/index.js

```diff
@@ -102,6 +104,13 @@ class LndEngine {
     this.retrieveWindowDuration = retrieveWindowDuration || 6000
     this.claimWindowDuration = claimWindowDuration || 30 * this.secondsPerBlock
 
+    // The amount of time, in seconds, that we'd like to buffer any output timelock
+    // by to account for block ticks during a swap This is especially problematic on
+    // regtest where we mine blocks every 10 seconds and is a known issue on
+    // mainnet.
+    // @see {@link https://github.com/lightningnetwork/lnd/issues/535}
+    this.blockBuffer = blockBuffer || 1200
```

Comment:
nit: Since this seems to have more to do with blocks maybe we could express it as `2 * this.secondsPerBlock`